python-client/giskard/push/contribution.py

```diff
@@ -103,9 +103,13 @@ def _create_text_contribution_push(
         ContributionPush: An object representing the contribution push for text features.
     """
     # Explain the text feature
+    input_df = sliced_ds.df
+    if model.meta.feature_names:
```

Comment:
why not calling `model.prepare_dataframe` that already this logic?

Reply:
You're right, thanks!